spec/topological_inventory/openshift/operations/core/authentication_retriever_spec.rb

```diff
@@ -5,9 +5,52 @@ module Openshift
     module Operations
       module Core
         RSpec.describe AuthenticationRetriever do
-          let(:subject) { described_class.new(endpoint.id) }
+          let(:subject) { described_class.new(123) }
+
+          let(:headers) { {"Content-Type" => "application/json"} }
+          let(:endpoint_authentications_url) do
+            "http://localhost:3000/api/topological-inventory/v0.0/endpoints/123/authentications"
+          end
+          let(:internal_authentications_url) do
+            "http://localhost:3000/api/internal/v0.0/authentications/321?expose_encrypted_attribute[]=password"
```

Comment:
The `/internal` api is not in the `/api` scope

Reply:
Even with the `ENV["PATH_PREFIX"]` set? So https://github.com/ManageIQ/topological_inventory-collector-openshift/pull/31/files#diff-cb36b54da2a9c126f845a1f8a056a0b4R15 needs to be changed to not include it?

Reply:
Yes, this is intentional.  If it were exposed at `#{ENV["TOPOLOGICAL_INVENTORY_URL"]}/#{ENV["PATH_PREFIX"]}/internal`, it would be publicly routable through the 3Scale gateway.  Since we don't use the prefix, the only way to access it is directly through the OpenShift service (not the gateway), so only callers from inside the cluster have access.